src/products/renderProducts.js

```diff
@@ -0,0 +1,215 @@
+        userViewOffers.id = `${offer.id}`;
+        userViewOffers.product = `${offer.product_name}`;
+        userViewOffers.farmName = `${offer.farm_name}`;
+        userViewOffers.grade = `${offer.grade}`;
+        userViewOffers.product_id = `${offer.product_id}`;
+        userViewOffers.availableUnits = `${offer.available_units}`;
+
+        if (offer.status === "0") {
+          offeringText += `KES ${offer.unit_price}`;
+          userViewOffers.unitPrice = `${offer.unit_price}`;
+          status[offer.id] = "unit";
+        } else if (offer.status === "1") {
+          offeringText += `KES ${offer.group_price}`;
+          status[offer.id] = "group";
+          userViewOffers.groupPrice = `${offer.group_price}`;
+        } else {
+          offeringText += `Group price: ${offer.group_price}, Unit Price: ${offer.unit_price}`;
+          userViewOffers.unitPrice = `${offer.unit_price}`;
+          userViewOffers.groupPrice = `${offer.group_price}`;
+          status[offer.id] = "both";
+        }
+        offersArray.push(userViewOffers);
+
+        client.set("offersArray", JSON.stringify(offersArray));
+      });
+      // offeringText += menus.viewCart;
+
+      message = `${con()} Choose one of the available options to buy. ${offeringText}`;
+    } else {
+      message = `${con()} Product not available`;
+      message += menus.footer;
+    }
+    return {
+      message,
+      status,
+    };
+  } catch (err) {
+    return err;
+  }
+};
+
+export const showAvailableProducts = async (client, textValue, text) => {
+  // Added client
+  if (textValue === 1) {
+    message = await renderProductCategories();
+  } else if (textValue === 2) {
+    const selection = parseInt(text.split("*")[1], 10);
+    message = await renderProducts(selection);
+  } else if (textValue === 3) {
+    const selection = parseInt(text.split("*")[2], 10);
+    const result = await renderOfferings(client, selection);
+
+    offeringStatus.push(result.status);
+
+    message = result.message;
+  } else if (textValue === 4) {
+    const selection = parseInt(text.split("*")[3], 10);
+    message = checkGroupAndIndividualPrice(
+      offeringStatus[`${offeringStatus.length - 1}`][`${selection}`]
+    );
+  } else if (textValue === 5) {
+    message = askForQuantity();
+  } else if (textValue === 6 && parseInt(text.split("*")[5], 10) > 0) {
+    const userQuantity = parseInt(text.split("*")[5], 10);
+    const id = text.split("*")[3];
+    const typeOfOffering = offeringStatus[offeringStatus.length - 1];
+    const selection = parseInt(text.split("*")[3], 10);
+    const purchasingOption = text.split("*")[4];
+    const availablePrice = typeOfOffering[`${selection}`];
+    const price = priceToUse(availablePrice, purchasingOption);
+    message = await confirmQuantityWithPrice(userQuantity, id, price, client);
+  } else if (textValue === 7 && text.split("*")[6] === "1") {
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/JoeMindo/USSDTest/pullRequest?prid=8695127)
```suggestion
  } else if (textValue === 7 && text.split('*')[6] === "1") {
```